kim_cnn/main.py

```diff
@@ -1,85 +1,76 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-import model
-from network import cnnTextNetwork
-from configurable import Configurable
-import torch
-import numpy as np
 import os
+import sys
+import numpy as np
+import torch
```

Comment:
By PEP8 standards,  put the imports in the following order: standard library imports, related third party imports, local application/library specific imports